plugins/status.js

```diff
@@ -0,0 +1,245 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const async = require('async');
+
+exports.register = function () {
+    this.outbound = require('./../outbound');
+    this.queue_dir = require('./../outbound/queue').queue_dir;
+}
+
+exports.hook_capabilities = function (next, connection) {
+
+    if (connection.remote.ip === '127.0.0.1' || connection.remote.ip === '::1') {
```

Comment:
Any reason not to use the already computed `is_local` flag?

```suggestion
    if (connect.remote.is_local) {
```

Reply:
FYI: the `is_local` property also takes into account the link-local (IPv4 and 6) and IPv6 unique local assignments, so it's a fair bit more robust.

Reply:
make sense